### PR TITLE
fix: [tag] the vault can tag info

### PR DESCRIPTION
**Pull request by GongHeng2017**
1. Abandoned hook_CanTag event
2. Add hook_CanMarked event
3. Vault not use tag function
4. change the tag key when virsual url.

Log: fix issue
Bug: https://pms.uniontech.com/bug-view-205873.html

### CODE REVIEW

File: src/plugins/common/dfmplugin-tag/tag.h [hunk 1]
```diff
@@ -20,13 +20,11 @@ class Tag : public dpf::Plugin
     DPF_EVENT_NAMESPACE(DPTAG_NAMESPACE)
     // slot events
     DPF_EVENT_REG_SLOT(slot_GetTags)
+    DPF_EVENT_REG_SLOT(slot_AddTagsBlackList)
 
     // signal events
     DPF_EVENT_REG_SIGNAL(signal_ReportLog_MenuData)
 
-    // hook events
-    DPF_EVENT_REG_HOOK(hook_CanTag)
```
Comment (max-lvs) [inline src/plugins/common/dfmplugin-tag/tag.h:28]: hook_CanTag 是模块对外接口，不能随意移除

File: src/plugins/common/dfmplugin-tag/tag.h [hunk 2]
```diff
@@ -25,7 +25,8 @@ class Tag : public dpf::Plugin
     DPF_EVENT_REG_SIGNAL(signal_ReportLog_MenuData)
 
     // hook events
-    DPF_EVENT_REG_HOOK(hook_CanTag)
+    DPF_EVENT_REG_HOOK(hook_CanTag) // Warning: The event is abandoned, please do not use it.
+    DPF_EVENT_REG_HOOK(hook_CanMarked)
```
Comment (pppanghu77) [inline src/plugins/common/dfmplugin-tag/tag.h]: Mark是机翻的标记吗，标记最好使用tag名称

File: src/plugins/common/dfmplugin-tag/tag.h (hunk 1 above)
Comment (GongHeng2017) [reply src/plugins/common/dfmplugin-tag/tag.h:28]: done

File: src/plugins/common/dfmplugin-tag/tag.h (hunk 2 above)
Comment (GongHeng2017) [reply src/plugins/common/dfmplugin-tag/tag.h]: done

**Review by pppanghu77 — APPROVED** [review]

**Review by max-lvs — APPROVED** [review]

**Pull request merged**